dhis-2/dhis-e2e-test/src/main/java/org/hisp/dhis/TestRunStorage.java

```diff
@@ -52,14 +51,14 @@ public static void addCreatedEntity( final String resource, final String id )
         createdEntities.put( id, resource );
     }
 
-    public static Map<String, String> getCreatedEntities()
+    public static LinkedHashMap<String, String> getCreatedEntities()
```

Comment:
Better to use Map (not implementation class). Code to interfaces, not implementations.